packages/node/src/meta/mmrQuery.controller.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020-2022 OnFinality Limited authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+  Controller,
+  Get,
+  HttpException,
+  HttpStatus,
+  Param,
+  Query,
+} from '@nestjs/common';
+import { MmrService } from '../indexer/mmr.service';
+import { getLogger } from '../utils/logger';
+
+const logger = getLogger('mmrs');
+
+@Controller('mmrs')
+export class MmrQueryController {
+  constructor(private mmrService: MmrService) {}
+
+  @Get(':blockHeight')
+  async getMmr(@Param() params) {
+    try {
+      // add await, otherwise error been skipped
+      return await this.mmrService.getMmr(params.blockHeight);
```

Comment:
I'm not sure what is the default way nestjs serialize a Uint8Array. Can you make the process more explicit?